### PR TITLE
ED-2111: Add OCI oss support in Build/Plugins/GenericEditor

### PR DESCRIPTION
**Pull request by ddevadat**
Add OCI oss support in Build/Plugins/GenericEditor

### CODE REVIEW

**Comment by ddevadat** [comment]
@pallakartheekreddy 

The circle ci code has some logic like below

```
prdata=$(curl -X GET -u $GITHUB_USER_TOKEN:x-oauth-basic https://api.github.com/repos/project-sunbird/sunbird-generic-editor/pulls/$CIRCLE_PR_NUMBER)
target_branch=$(echo "${prdata}" | jq -r '.base.ref')
git clone https://github.com/project-sunbird/sunbird-content-plugins.git plugins -b $target_branch
```

Based on the circle ci error
```
Cloning into 'plugins'...
fatal: Remote branch null not found in upstream origin

Exited with code exit status 128

```

it looks like the variable target_branch is getting evaluated as null.  Could you please explain the expected behaviour here.

e.g if the PR branch is **branch_a** for generic editor, does it expect to clone content-plugin from upstream **branch_a** ?

File: app/bower.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
         "oclazyload": "^1.0.9",
         "ngSafeApply": "./libs/Scope.SafeApply.js",
         "izitoast": "^1.1.3",
-        "contenteditor": "https://sunbirdstagingpublic.blob.core.windows.net/sunbird-content-staging-knowlg/content-editor/scripts/base-editor.min.js"
+        "contenteditor": "https://objectstorage.ap-mumbai-1.oraclecloud.com/n/bmzbbujw9kal/b/odev-dev-diksha-contents/o/content-editor/scripts/base-editor.min.js"
```
Comment (pallakartheekreddy) [inline app/bower.json]: @ddevadat Please revert this change. This OCI specific change not required in sunbird.

Comment (ddevadat) [reply app/bower.json]: will look into this

Comment (ddevadat) [reply app/bower.json]: @pallakartheekreddy its removed. please review

Comment (pallakartheekreddy) [reply app/bower.json]: This PR is changes not required in Sunbird
